src/help-extension/index.ts

```diff
@@ -45,9 +41,32 @@ import {
   CommandIDs as StateDBCommandIDs
 } from '../statedb';
 
-import {
-  CommandIDs
-} from './';
+
+/**
+ * The command IDs used by the help plugin.
+ */
+namespace CommandIDs {
+  export
+  const open: string = 'help-jupyterlab:open';
```

Comment:
+1 to having these unexported consts in a single namespace. I don't think we should type them as strings, though. The inferred type (the actual string) is better, I think, as it allows comparison, etc.